File: DuDoong-Domain/src/main/java/band/gosrock/domain/domains/event/exception/AlreadyCloseStatusException.java
package band.gosrock.domain.domains.event.exception;


import band.gosrock.common.exception.DuDoongCodeException;

public class AlreadyCloseStatusException extends DuDoongCodeException {

    public static final DuDoongCodeException EXCEPTION = new AlreadyCloseStatusException();

    private AlreadyCloseStatusException() {
        super(EventErrorCode.ALREADY_CLOSE_STATUS);
    }
}

File: DuDoong-Domain/src/main/kotlin/band/gosrock/domain/domains/event/exception/AlreadyCloseStatusException.kt
package band.gosrock.domain.domains.event.exception

import band.gosrock.common.exception.DuDoongCodeException

class AlreadyCloseStatusException private constructor() : DuDoongCodeException(EventErrorCode.ALREADY_CLOSE_STATUS) {
    companion object {
        @JvmField
        val EXCEPTION: DuDoongCodeException = AlreadyCloseStatusException()
    }
}
